body.tex

```diff
@@ -0,0 +1,52 @@
+\setActionItemPrefix{S-}
+
+\section{Actions (temporary) }
+This section should disappear when these are all done.
+\listofactions
+
+\section{Introduction}
+This is the Rubin \gls{DP} information and data security plan for operations.
+This renders \citeds{LDM-324} obsolete.
+
+This security plan conforms with the construction era security documentation:
+\begin{itemize}
+\item “LSST Information Classification Policy” \citedsp{LPM-122}
+\item “LSST Master Information Security Policy” \citedsp{LPM-121}
+\end{itemize}
+
+Not has been taken of \cite{TCISSG}.
+
+During the ramp up to operations further security requirements were given to the project by the agencies.
+The response to these is in \gls{DMTN}-199\footnote{This is a limited availability document}.
+
+The \gls{DP} department is concerned with the operations and maintenance  of Rubin constructed software, hardware and networks.
+Development remains open-source in nature.
+
+Data from the telescope on Cerro Pach\'{o}n is transmitted over Rubin controlled networks to \gls{SLAC} in  Menlo Park USA.
+After an agreed embargo period all data will be transmitted to the \gls{IN2P3} in Lyon, France, and at least some data is transmitted to the Royal Observatory Edinburgh.
+Processing occurs in all three locations using a coordinate release of the Rubin \gls{Science Pipelines}.
+
+Access to data will be provided to authorized via a US \gls{DAC}
+A data access center similar to the \gls{US} \gls{DAC} will also be provided in Chile.
+
+For data taken initially we have been requested to comply with \citeds{NIST.SP.800-171} the compliance matrix is provided in \appref{ap:compliance}
+
+
+This document does not do a detailed threat analysis though this should be and are done for individual systems and \gls{Enclave}s. \cite{TCISSG} is good guide for software threats.
```

Comment:
1. Not sure what that TCISSG citation is
2. I think this risks saying that the plan is to have a plan. Maybe we can define a rubrik for each enclave to describe its arrangements.

Reply:
ok we can discuss - its pretty endless though